utility/test_repoutil.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2019 Cisco and its affiliates
```

Comment:
Copyrigh has changed see [here](https://github.com/YangCatalog/backend/blob/master/parseAndPopulate/parseException.py)

No Cisco copyright for new files. IETF only